### ISSUE TITLE
Pokemobs randomly get other's nicknames.

### ISSUE THREAD

**Issue by Thutmose** [Thutmose/Pokecube#143]
#### Issue Description:

Sometimes Pokemobs will randomly aquire the nickname of another pokemob on the server.
#### What happens:

Pokemob's nickname changes to something else.
#### What you expected to happen:

Mobs not change nickname.
#### Steps to reproduce:

Currently known, needs testing.

---
#### Affected Versions (Do _not_ use "latest"):
- Pokecube Core: 4.6.1
- Pokecube Revival (if applicable): 4.5.0
- Pokecube Mobs (if applicable): 2.2.3
- Minecraft: 1.10.2
- Forge: 2075

**Comment by Disturbeddragon1** [comment]
Yeah it's been happening to us as well, and we've only been playing local Lan between the two of us. It definitely seems random though cause not all Pokemobs will acquire the nickname of another Pokemob. Seems like it's the newest nickname though that pops up. So if someone names their Pokemob "Paladin", then names another Pokemob "Hunter", it will random nickname a wild Pokemob the latter.

Also running the following:

Affected Versions (Do not use "latest"):

```
Pokecube Core: 4.6.1
Pokecube Revival (if applicable): 4.5.0
Pokecube Mobs (if applicable): 2.2.3
Minecraft: 1.10.2
Forge: 12.18.1.2087
```

**Comment by Thutmose** [comment]
I think this should have been fixed in 4.8.0, please let me know if you see it occur again.

**Comment by Thutmose** [comment]
this has not been reported since 4.8.0, so i will close this issue now.
